src/core/wb.js

```diff
@@ -57,9 +57,33 @@ var getUrlParts = function( url ) {
 
 	/**
 	 * @variable i18n
-	 * @return {string} of HTML document language
+	 * @return {string} of WET document language
```

Comment:
Would this count as a major change?

If so, should this PR be reworked to use something along the lines of a ``wb.wbLang`` variable and make WET use it instead of ``wb.lang``?

Reply:
As long we don't need to modify existing code that call `wb.lang` don't need to pass a parameter.